pkgs/servers/mastodon/default.nix

```diff
@@ -22,14 +21,6 @@ stdenv.mkDerivation rec {
     sha256 = "sha256-Ngfs15YKLfSBOKju3BzpZFnenB370jId2G1g9Qy1y5w=";
```

Comment:
This also needs to be updated.
https://github.com/NixOS/nixpkgs/pull/167946/commits/08e700c531fd31b3a96e61a31e7b0c3af5ff3ad2 - Can you add this commit?